docs/spec_draft.md

```diff
@@ -927,6 +928,47 @@ spaces of `result` and `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reverse
+
+### Semantics
+
+Reverses the order of elements in the `operand` along the specified `dimensions`, generating an output array of the same shape. Each element of the operand array at a multidimensional index is stored into the output array at a transformed index. The multidimensional index is transformed by reversing the index in each dimension to be reversed (i.e., if a dimension of size N is one of the reversing dimensions, its index i is transformed into N - 1 - i).
```

Comment:
"generating an output array of the same shape". So far, the convention has been to write this as "and produces a \`result\` tensor".